test/destroy-test.js

```diff
@@ -66,6 +66,7 @@ makeTest('test destroy() cleans and removes only leveldb parts of a dir', functi
     t.notOk(err, 'no error')
     leveldown.destroy(location, function () {
       readfiletree(location, function (err, tree) {
+        t.notOk(err, 'no error')
```

Comment:
> should we use t.error for all these?

Yea, that has the benefit that it prints the error message if there is one. No biggie